src/style/globals.css

```
@@ -1,6 +1,28 @@
+@import '../sd/build/css/_variables.css';
+
+* {
+  box-sizing: border-box;
+}
+
+*::-webkit-scrollbar {
+  width: 0.3rem;
+  height: 0.3rem;
+}
+*::-webkit-scrollbar-thumb {
+  background-color: theme('colors.gray.85');
```

Biome (1.9.4)

[error] Unexpected unknown function: theme

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)